Readme.md

```diff
@@ -1,18 +1,44 @@
-# gdal
+# GDAL
 
 [![Documentation](https://docs.rs/gdal/badge.svg)](https://docs.rs/gdal)
 ![Build Status](https://github.com/georust/gdal/workflows/CI/badge.svg)
 
-[GDAL](http://gdal.org/) bindings for [Rust](http://www.rust-lang.org/).
+[GDAL](http://gdal.org/) is a translator and processing library for various raster and vector geospatial data formats. 
 
-So far, you can:
+This crate provides safe, idiomatic [Rust](http://www.rust-lang.org/) bindings for GDAL.
 
-* open a raster dataset for reading/writing
-* get size and number of bands
-* get/set projection and geo-transform
-* read and write raster data
-* warp between datasets
-* read and write vector data
-* access metadata
+## Capabilities
 
-Many raster and vector functions are not available. Patches welcome :)
+GDAL is an incredibly powerful library. For a general understanding of its capabilities, a good place to get started is the [GDAL User-oriented documentation](https://gdal.org/user/index.html). These features include:
+
+* Open a raster dataset for reading/writing
+* Get the raster size and number of bands
+* Get and set the projection and geo-transform
+* Warp (resample and reproject) between datasets
+* Read and write vector data
+* Read and write metadata
+* Translate between formats
+
+## Documentation
+
+This crate's [API documentation](https://docs.rs/crate/gdal) is hosted on [docs.rs](https://docs.rs). The Rust documentation is currently a work in progress, and may not cover requisite details on parameter semantics, value interpretation, etc.. Therefore, the authoritative documentation for the [GDAL C API](https://gdal.org/api/index.html#c-api) may be a helpful reference.
+
+## Usage
+
+This crate provides idiomatic Rust bindings for the GDAL C library. To do that, it internally uses [`gdal-sys`](gdal-sys), which is a  [`bindgen`](https://rust-lang.github.io/rust-bindgen/)-generated sub-crate, rendered directly from the GDAL C library. Normally one would not need to interact with the `gdal-sys` crate. However, not every single feature of the GDAL C API has been fully exposed, so in some cases the `gdal-sys` crate may be necessary to accessing more esoteric features of GDAL.
```

Comment:
```suggestion
This crate provides high-level, idiomatic Rust bindings for GDAL.
To do that, it internally uses [`gdal-sys`](gdal-sys), a  low-level interface to the GDAL C library, generated using [`bindgen`](https://rust-lang.github.io/rust-bindgen/).
Using the `gdal-sys` crate is normally not needed, but it can be useful in order to call APIs that have not yet been exposed in `gdal`.
```